os/src-jvm/ProcessOps.scala

```diff
@@ -22,6 +22,8 @@ import scala.annotation.tailrec
   *   want
   */
 case class proc(command: Shellable*) {
+  lazy val commandChunks = command.flatMap(_.value)
```

Comment:
Is it necessary to persist it? Can't it be a simple `def`?

Reply:
I don't have a strong opinion on that, other than that it appears very sensible that os.proc() provides access to the actual command (for the same rationale that it is sensible that CommandResult not only contains the result but the actual command that caused the result).
I used lazy here because it appeared to be the idiomatic thing to do, even now that it's not a big calculation. And if the overhead of the synchronization imposed by lazily evaluation `commandChunks` becomes a bottleneck in an app, then the app has a fundamental design issue in that region. But that said, I am nowhere near to decide if `lazy` or `def` is more idiomatic in such cases and I am happy to hear the opinion of Scala veterans on this.

Reply:
This is pure convenience, as `command` is already accessible, and easy to compute, so it's IMHO not worth the extra memory space, hence `def` is better.

Reply:
Fair point, switched to `def`.